app/src/main/res/values-de/strings.xml

```diff
@@ -549,11 +549,20 @@ Signatur: %5$s</string>
     <string name="discard_changes_question">Änderungen verwerfen?</string>
     <string name="discard">Verwerfen</string>
     <string name="keep_editing">Weiter bearbeiten</string>
+
+    <string name="channel_description_general">Nächste Vorlesung, News, Erinnerung über Studienbeiträge</string>
```

Comment:
either close with etc. / usw. or add a conjunction for the last element

Reply:
You’re right. Added a conjunctin.